GittieLabs/lovebank-frontend
# Feature three pager

Cycles through the three pages on touch.
    Generally well positioned elements
    Only tested on Android emulator

Test instructions:
- Verify code quality
- Verify code organization
- Verify descriptive comments
- Verify graphics on both IPhone emulator and Android emulator
- Verify design details
- Verify any other assorted concerns omitted here
